src/App.js

```diff
@@ -1,13 +1,66 @@
 import React from "react";
 import "./App.css";
-
 class App extends React.Component {
+constructor(){
+  super()
+  this.state = { 
+    currentScore: 0,
+    AddScore: 1,
```

Comment:
```suggestion
    addScore: 1,
```
the way you capitalized "AddScore" is unconventional for a key in `this.state`. This form of casing is called "Pascal case" and should be reserved for things like components. Instead, use "camel case". Here's an article on the difference between the two: https://www.theserverside.com/answer/Pascal-case-vs-camel-case-Whats-the-difference